packages/frontend/src/providers/DashboardProvider.tsx

```diff
@@ -102,7 +102,8 @@ const Context = createContext<DashboardContext | undefined>(undefined);
 
 export const DashboardProvider: React.FC<{
     schedulerFilters?: SchedulerFilterRule[] | undefined;
-}> = ({ schedulerFilters, children }) => {
+    dateZoom?: DateGranularity | undefined;
```

Comment:
we need to pass dateZoom to the provider to pick it up on load, not on use effect, otherwise minimal chart will not update.